lib/components/app/app-menu.js

```diff
@@ -41,9 +47,93 @@ class AppMenu extends Component {
     window.location.href = startOverUrl
   }
 
+  _expandPane = () => {
+    const { isPaneOpen } = this.state
+    this.setState({ isPaneOpen: !isPaneOpen })
+  }
+
+  _expandSubmenu = (id) => {
+    const { expandedSubmenus } = this.state
+    const currentlyOpen = expandedSubmenus[id] || false
+    this.setState({ expandedSubmenus: { [id]: !currentlyOpen } })
+  }
+
+  _addExtraMenuItems = (menuItems) => {
+    const AddIconAndLabel = ({iconType, iconUrl, label}) => {
+      return (
+        iconUrl
+          ? <span>
+            <img alt={`Icon for ${label} menu item`} src={iconUrl} />
+            {label}
+          </span>
+          : <span><Icon name={iconType || 'external-link-square'} />{label}</span>
+      )
+    }
+
+    return (
+      menuItems && menuItems.map(menuItem => {
+        const {
+          children,
+          href,
+          iconType,
+          iconUrl,
+          id,
+          label,
+          subMenuDivider
+        } = menuItem
+        const { expandedSubmenus } = this.state
+        const isSubmenuExpanded = expandedSubmenus[id]
+
+        if (children) {
+          return <>
+            <MenuItem
+              className='expansion-button-container menu-item'
+              header
+              key={id}>
```

Comment:
Move `>` to new line.

Reply:
Prettier run in 542d756098ceae9ad56f74334fae242b4be7c938